synfig-studio/src/synfigapp/canvasinterface.cpp

```diff
@@ -699,6 +699,45 @@ CanvasInterface::jump_to_prev_keyframe()
 	//catch(...) { synfig::warning("Unable to find prev keyframe"); }
 }
 
+static void update_layer_size(CanvasInterface& canvas_interface, Layer::Handle& layer, bool resize_image) {
+	int w, h;
+
+	w = layer->get_param("_width").get(int());
+	h = layer->get_param("_height").get(int());
```

Comment:
If you'll make another commit/push (for changing the method parameter list), what about declaring and initializing in the same line?
`int w = layer-> ...`

Reply:
Done